src/Servers/Kestrel/Core/src/Http3Limits.cs

```diff
@@ -33,25 +32,5 @@ internal int HeaderTableSize
                 _headerTableSize = value;
             }
         }
-
-        /// <summary>
-        /// Indicates the size of the maximum allowed size of a request header field sequence. This limit applies to both name and value sequences in their compressed and uncompressed representations.
-        /// <para>
-        /// Value must be greater than 0, defaults to 8192
-        /// </para>
-        /// </summary>
-        public int MaxRequestHeaderFieldSize
```

Comment:
I think this can be configured on the server with [MaxRequestHeadersTotalSize](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.server.kestrel.core.kestrelserverlimits.maxrequestheaderstotalsize?view=aspnetcore-5.0).

Reply:
I think this makes sense. I wonder if we couldn't do the same for HTTP/2 and deprecate Http2Limits.MaxRequestHeaderFieldSize. For HTTP/2, It's used by HPackDecoder, but I don't really see the harm in using MaxRequestHeadersTotalSize for the same purpose.

Reply:
They actually have different purposes. total size = total header section size. This property is specific to an individual header. I rolled the changes I had related to this back.

Reply:
Yeah. And that's why we probably won't change it. I just think this was something we never needed to expose. 
I just think it was too fine grain of a nob to expose. We don't enforce a per-field or per-header limit for HTTP/1.x either. I don't see the harm in making the MaxRequestHeadersTotalSize also the max field size. Our defaults are pretty low anyway.